Automattic/sensei
# Rename Course Theme to Learning Mode

Fixes #5580 

### Changes proposed in this Pull Request

* Changes every reference of `course theme` to `learning-mode`, this including 'Course Theme', 'Course_Theme' and other variations.
* Changes the course meta `_course_theme` to `_learning_mode` as well as a mini migration of the former value. 
* Renames all files and directories.
* Renames CSS classes and JS variables.
* Leaves an alias of affected PHP class names.

### Testing instructions
This is a big one.

- Test thoroughly all learning mode features. Test enabling and disabling learning mode.
- Run the tests locally to double check: `npm run test`.
